core/bentley/src/ProcessDetector.ts

```diff
@@ -54,6 +54,11 @@ export class ProcessDetector {
   */
   public static get isMobileBrowser() { return this.isIOSBrowser || this.isAndroidBrowser; }
 
+  /** Is this process running in a Chrome or Edge browser or an Electron front end? */
+  public static get isChromeEdgeElectron() {
+    return (this.isBrowserProcess && window.navigator.userAgent.indexOf("Chrome") > -1 && window.navigator.userAgent.indexOf("OP") === -1) || this.isElectronAppFrontend;

```

Comment:
Consider changing this to something like `isChromium`. It's going to return true for most (all?) chromium-based browsers and false for old Edge.

Reply:
Done